charts/kong/templates/controller-service-account.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ template "kong.serviceAccountName" . }}
+  annotations:
+  {{- if .Values.ingressController.serviceAccount.annotations }}
+{{ toYaml .Values.ingressController.serviceAccount.annotations | indent 4 }}
+  {{- end }}
   namespace: {{ .Release.Namespace }}
```

Comment:
Can we please keep `namespace` and `name` together?
Please move the new code below `namespace` line.

Reply:
Make sense to me :100: